app/src/components/Home/CoverPage.jsx

```diff
@@ -16,11 +16,24 @@ class CoverPage extends Component {
 
   constructor(props) {
     super(props);
+
     this.state = {
-      currentSlider: 0
+      currentSlider: 0,
+      autoPlaySlider: true,
+      speedPlaySlider: 10000
     };
   }
 
+  componentDidUpdate() {
+    $(`.${CoverPageStyle['dots-cover']}`).off('click').on('click', () => {
+      if (!this.state.autoPlaySlider) return;
+
+      this.state = {
```

Comment:
🚨 Use `this.setState({ autoPlaySlider: false });`. You can't directly mutate the state.